solidstart-tanstackquery-tailwind-modules/src/components/PRAndIssueCard/PRAndIssueCard.tsx

```diff
@@ -0,0 +1,137 @@
+import { For, Match, Show, Switch, splitProps } from 'solid-js';
+import { format } from 'date-fns';
+import cn from 'classnames';
+import {
+  IssuesIcon,
+  PullRequestIcon,
+  MergedPrIcon,
+  ClosedPrIcon,
+  CommentIcon,
+  ClosedIssueIcon,
+} from '../Icons';
+import { A } from '@solidjs/router';
+
+export type Labels = {
+  color?: string;
```

Comment:
Is color optional? I think all labels must have color right?

Reply:
yeah true. There was an issue. it's not optional